File: integrations/aws-strands/error-code-corpus/typescript/code_through_local.ts
/** A code reached through a local binding rather than written at the site. */
import { EventType, type BaseEvent } from "@ag-ui/core";

export function emit(): BaseEvent {
  const failureCode = "THROUGH_LOCAL";
  return {
    type: EventType.RUN_ERROR,
    message: "Resolved through a local.",
    code: failureCode,
  };
}

File: integrations/aws-strands/error-code-corpus/python/code_through_local.py
"""A code reached through a local binding rather than written at the site."""

from ag_ui.core import EventType, RunErrorEvent


def emit() -> RunErrorEvent:
    failure_code = "THROUGH_LOCAL"
    return RunErrorEvent(
        type=EventType.RUN_ERROR,
        message="Resolved through a local.",
        code=failure_code,
    )
